src/ElasticsearchService.php

```diff
@@ -24,25 +23,26 @@ public function __construct(Client $client)
         $this->client = $client;
     }
 
-
     /**
      * @inheritdoc
      */
     public function search(array $params = [])
     {
+        $params = IndexNamePrefixer::getPrefixedIndexParameters($params);
```

Comment:
Do we have other kind of `Prefixer`? Should we inject this via constructor or make it as a trait?

Reply:
This is the only prefixer, but it's used a bit here and there that's why it's in a separate class

Reply:
I'd also like to not change the public API, so injection may be a bit tricky